### PR TITLE
Update documentation related to Google reCAPTCHA

### PR DESCRIPTION
**Pull request by sdzhepa**
## Purpose of this pull request

This pull request (PR) updates user documentation related to Google reCAPTCHA to reflect all changes for old and newly added functionality during a couple of previous releases

## Affected documentation pages

- https://docs.magento.com/m2/ce/user_guide/stores/security-google-recaptcha.html
- https://docs.magento.com/m2/ce/user_guide/configuration/security/google-recaptcha.html

## Affected Magento editions

- [x] Open Source
- [ ] Commerce
- [ ] B2B

**Note**: _this functionality exists in Open Source, Commerce, and B2B Magento editions without any differences_

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by sdzhepa** [comment]
Hello @keharper 

Could you please assist with the review?

File: src/stores/security-google-recaptcha.md [hunk 1]
```diff
@@ -7,17 +7,21 @@ Google reCAPTCHA ensures that a human being, rather than a computer, is interact
...
+[Google reCAPTCHA][1] verifies users with increased security and reduced clicks for each of completion, which can be implemented in several ways:
 
-- reCAPTCHA v2 verifies users and customers using an on-click feature proving _I am not a robot_.
-- Invisible reCAPTCHA (**recommended**) performs background verification without user interaction. Users and customers are automatically verified without clicking anything.
+1. reCAPTCHA v2  verifies users and customers using an on-click feature and includes two types
 
-You can use Google reCAPTCHA on the [Admin Sign In]({% link stores/admin-signin.md %}) page, and on various customer pages in the storefront. The styling options include your choice of a light or dark theme, and either a compact or normal size.
+    - _"I am not a robot"_ Checkbox Validate requests with the "I'm not a robot" checkbox. Users and customers  need to click the checkbox and possibly select specific images to complete a challenge.
+    - Invisible reCAPTCHA badge (**recommended**) performs background verification without user interaction. Users and customers are automatically verified and possibly select specific images to complete a challenge.
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
    - Invisible reCAPTCHA badge (**recommended**) performs background verification without user interaction. Users and customers are automatically verified, but might be required to select specific images to complete a challenge.
```

File: src/stores/security-google-recaptcha.md [hunk 2]
```diff
@@ -7,17 +7,21 @@ Google reCAPTCHA ensures that a human being, rather than a computer, is interact
...
 
-If you have the standard [Magento CAPTCHA]({% link stores/security-captcha.md %}) enabled, reCAPTCHA will override it. You can have both enabled on your Magento instance without issues.
+2. reCAPTCHA v3 Verify requests with a score based on Google algorithm.
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
2. reCAPTCHA v3 Verify requests with a score based on a Google algorithm.
```

File: src/stores/security-google-recaptcha.md [hunk 3]
```diff
@@ -28,15 +32,21 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
+   For example, enter `Magento v2 Invisible`. You only need one set of keys per each reCAPTCHA type for your entire Magento instance.
 
 1. Choose the type of Google reCAPTCHA that you want to use:
 
-    - `reCAPTCHA V2` — Verifies with a button click.
-    - `Invisible reCaptcha` — Verifies in the background without interaction.
-
+    - `reCAPTCHA v3` — Verify requests with a score
+        
+    - `reCAPTCHA v2` — Verifies with a button click or images selection
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
    - `reCAPTCHA v2` — Verifies with a button click or image selections
```

File: src/stores/security-google-recaptcha.md [hunk 4]
```diff
@@ -51,41 +61,47 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
-
-    - When the domain(s) you added appear at the top of the page, choose the Magento domain to get your keys.
-
-    - Scroll down to the **Adding reCAPTCHA to your site** section and expand **Keys**. You will need these keys when enabling Google reCAPTCHA.
+1. Click **SUBMIT** to complete registration and receive keys.
 
     ![Google reCAPTCHA - keys]({% link images/images/recaptcha-keys.png %})
+    
+{:.bs-callout-info}
+**Pay attention!** Not all keys are applicable for all types of reCAPTCHA and could lead to unexpected behavior. Example Google reCAPTCHA keys generated for _reCAPTCHA v2 "I'm not a robot"_ will not work with _reCAPTCHA v2 Invisible_ and could block functionality where reCAPTCHA is enabled.
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
**Important!** Not all keys are applicable for all types of reCAPTCHA, and misapplying them could lead to unexpected behavior. For example, Google reCAPTCHA keys generated for _reCAPTCHA v2 "I'm not a robot"_ will not work with _reCAPTCHA v2 Invisible_ and could block functionality where reCAPTCHA is enabled.
```

File: src/stores/security-google-recaptcha.md [hunk 5]
```diff
@@ -51,41 +61,47 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
 1. In the upper-right corner, set **Store View** to `Default Config`.
 
-1. In the left panel, expand _Security_ and choose **Google reCAPTCHA**.
+1. In the left panel, expand _Security_ and choose **Google reCaptcha**.
 
-1. On the page, expand the **General** section and enter the following generated keys:
+1. On the page, expand the **General** section and enter the following generated keys for specific reCAPTCHA type that should be enabled:
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
1. On the page, expand the **General** section and enter the following generated keys for the specific reCAPTCHA type to be enabled:
```

File: src/stores/security-google-recaptcha.md [hunk 6]
```diff
@@ -51,41 +61,47 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
 
     - Google API website key
     - Google API secret key
 
     ![Google reCAPTCHA - keys configuration]({% link images/images/config-security-recaptcha-keys.png %}){: .zoom}
     [_Google reCAPTCHA keys_]({% link configuration/security/google-recaptcha.md %})
+    
+1. Select reCaptcha v2 type based on generated keys
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
1. Select the reCaptcha v2 type based on the generated keys    
```

File: src/stores/security-google-recaptcha.md [hunk 7]
```diff
@@ -99,14 +115,72 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
 
     - **Enable** — Set to `Yes`.
 
-    - **reCAPTCHA type** — Choose the type you want used in Admin forms. You must have added the correct API keys for the type.
+    - **Invisible badge position** — Choose `Inline` or `Bottom Right` or `Bottom Left`. Applicable only to _Invisible reCaptcha v2_ and _Invisible reCaptcha v3_.
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
    - **Invisible badge position** — Choose `Inline`, `Bottom Right`, or `Bottom Left`. Applicable only to _Invisible reCaptcha v2_ and _Invisible reCaptcha v3_. 
```

File: src/stores/security-google-recaptcha.md [hunk 8]
```diff
@@ -99,14 +115,72 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
+      - **Use in Create user**
+      - **Use in Review**
+      - **Use in PayPal PayflowPro payment form**
+      - **Use invisible ReCaptcha in newsletter** Requires an Invisible ReCaptcha v2 or v3 key. If enabled, a badge will be displayed in every page.
+      - **Use in Send To Friend**
+
+1. When complete, click <span class="btn">Save Config</span>.
+
+
+### For reCaptcha v3 type
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
### For reCaptcha v3
```

File: src/stores/security-google-recaptcha.md [hunk 9]
```diff
@@ -99,14 +115,72 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
+
+    ![Google reCAPTCHA v2 - type configuration]({% link images/images/config-security-recaptcha-type-v3.png %}){: .zoom}   
+
+1. Expand (![Expansion selector]({% link images/images/btn-expand.png %})) the **Backend** section and set the options
+
+   If needed, clear the **Use system value** checkbox to change the setting.
+
+    - **Enable** — Set to `Yes`.
+
+    - **Minimum score** — Input value from `0.0` to `1.0` where `1.0` is very likely a good interaction, `0.0` is very likely a bot. For more details see official documentation [Interpreting the score][3]
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
    - **Minimum score** — Input a value from `0.0` to `1.0`, where `1.0` is very likely a good interaction, and `0.0` is very likely a bot. For more, details see the official product documentation [Interpreting the score][3]   
```

File: src/stores/security-google-recaptcha.md [hunk 10]
```diff
@@ -99,14 +115,72 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
+
+    ![Google reCAPTCHA - frontend configuration]({% link images/images/config-2fa-google-recaptcha-v3-frontend.png %}){: .zoom}
+    [_Google reCAPTCHA Frontend_]({% link configuration/security/google-recaptcha.md %})
+
+   If needed, clear the **Use system value** checkbox to change the setting.
+
+    - **Enable** — Set to `Yes`.
 
-    - **Theme** — Choose `Light` or `Dark`.
+    - **Invisible badge position** — Choose `Inline` or `Bottom Right` or `Bottom Left`. Applicable only to _Invisible reCaptcha v2_ and _Invisible reCaptcha v3_.
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
    - **Invisible badge position** — Choose `Inline`, `Bottom Right`, or `Bottom Left`. Applicable only to _Invisible reCaptcha v2_ and _Invisible reCaptcha v3_. 
```

File: src/stores/security-google-recaptcha.md [hunk 11]
```diff
@@ -99,14 +115,72 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
...
+
+   If needed, clear the **Use system value** checkbox to change the setting.
+
+    - **Enable** — Set to `Yes`.
 
-    - **Theme** — Choose `Light` or `Dark`.
+    - **Invisible badge position** — Choose `Inline` or `Bottom Right` or `Bottom Left`. Applicable only to _Invisible reCaptcha v2_ and _Invisible reCaptcha v3_. 
 
-    - **Size** — Choose `Normal` or `Compact`.
+    - **Minimum score** — Input value from `0.0` to `1.0` where `1.0` is very likely a good interaction, `0.0` is very likely a bot. For more details see official documentation [Interpreting the score][3]
```
Comment (keharper) [inline src/stores/security-google-recaptcha.md]:
```suggestion
    - **Minimum score** — Input a value from `0.0` to `1.0`, where `1.0` is very likely a good interaction, and `0.0` is very likely a bot. For more details, see the official product documentation [Interpreting the score][3]    
```

**Comment by keharper** [comment]
@sdzhepa I can't push edits directly in this PR. There's one remaining question, and multiple grammatical edits.

**Comment by sdzhepa** [comment]
Hello @keharper 

Thank you very much for the review!

I have accepted all your suggested changes and fixed type with default re Captcha type(it should be V3).

Please, let me know if I missed something.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Running tests

**Comment by keharper** [comment]
Running tests

**Pull request merged**